charts/osm/values.yaml

```diff
@@ -166,13 +229,16 @@ osm:
   enableReconciler: false
 
   # -- Deploy Prometheus with OSM installation
-  deployPrometheus: false
+  deployPrometheus: true
```

Comment:
What's the reasoning for changing the default values for deploying Prometheus, Grafana, Jaeger, and fluentbit?

Reply:
A mistake. Fixed